.github/workflows/build.yaml

```diff
@@ -20,12 +20,17 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 20
     permissions:
-      contents: read
-      packages: write
+       contents: read
+       packages: write
+       pull-requests: write
+       actions: write
```

Comment:
required for plugins t o works, so they are able to set status checks and write comments to PRs